src/main/java/io/appium/java_client/remote/AndroidMobileCapabilityType.java

```diff
@@ -276,4 +373,54 @@
      * Parallel Testing Setup Guide</a> for more details.
      */
     String SYSTEM_PORT = "systemPort";
+
+    /**
+     * Optional remote ADB server host.
+     */
+    String REMOTE_ADB_HOST = "remoteAdbHost";
+
+    /**
+     * Skips unlock during session creation. Defaults to {@code false}
+     */
+    String SKIP_UNLOCK = "skipUnlock";
+
+    /**
+     * Unlock the target device with particular lock pattern instead of just waking up the device with a helper app.
+     * It works with {@code unlockKey} capability. Defaults to undefined. {@code fingerprint} is available only for
+     * Android 6.0+ and emulators.
+     * Read <a href="https://github.com/appium/appium-android-driver/blob/master/docs/UNLOCK.md">unlock doc</a> in
+     * android driver.
+     */
+    String UNLOCK_TYPE = "unlockType";
+
+    /**
+     * A key pattern to unlock used by {@code unlockType}.
+     */
+    String UNLOCK_KEY = "unlockKey";
+
+    /**
+     * Initializing the app under test automatically.
+     * Appium does not install/launch the app under test if this is {@code false}. Defaults to {@code true}
```

Comment:
it does install the app, but skips the launch

Reply:
@mykola-mokhnach I've changed description to 'Appium does not launch the app...'. Should it be also changed in http://appium.io/docs/en/writing-running-appium/caps/ ?

Reply:
yes, it makes sense to update the description in the documentation